doc/04-Templates.md

```diff
@@ -33,6 +33,10 @@ templates of its parent folders.
 Templates are organized within simple INI files. However, it is perfectly valid
 to define multiple templates in a single file.
 
+> **Note:**
+>
+> Template file will be ignored if the [graph] or [metric_filters] section is missing.
+
```

Comment:
To enhance the flow of reading, I'd put that at the end of this paragraph. (right after the *topics* get introduced)